lib/elixir/lib/code.ex

```diff
@@ -599,7 +602,10 @@ defmodule Code do
       "Converts an atom to a char list."
 
   """
-  def get_docs(module, kind) when is_atom(module) do
+  @doc_kinds [:docs, :moduledoc, :callback_docs, :type_docs]
+  def get_docs(module, kind \\ :all)
+
+  def get_docs(module, kind) when is_atom(module) and kind in @doc_kinds or kind == :all do
```

Comment:
good point. now that we removed the checked in the private function we can do it.